project2-beat-that/script.js

```diff
@@ -1,4 +1,272 @@
+// Generate 2 dice rolls for player 1
+// > return user the 2 dice rolls
+// > Allow player 1 to choose order of 2 dice rolls
+// > return the player's choice
+
+// 2nd version
+
+var comparingCombineDiceRolls = function () {
+  var max = Math.max.apply(null, playerScores);
+  var winningPlayer = playerScores.indexOf(max);
+  return winningPlayer;
+};
+
+var randomDiceRolls = function () {
+  return Math.floor(Math.random() * 6) + 1;
+};
+
+var diceRoll1 = 0;
+var diceRoll2 = 0;
+
+var rollState = true;
+var playerNumber = 0;
+var numPlayers = 2;
+var playerScores = Array(numPlayers);
+playerScores.fill(0);
```

Comment:
nice and clear global variables!

Reply:
nice implementation of the alternate way to create an array that we didnt cover 

you can actually do `var playerScores = Array(numPlayers).fill(0)` all in one line